hazelcast/src/main/java/com/hazelcast/internal/longregister/client/codec/LongRegister.yaml

```diff
@@ -0,0 +1,212 @@
+id: 255
```

Comment:
Should we keep this file at protocol repo as internal hidden?

Reply:
@asimarslan requested me to remove it from the official protocol repo. So I decided to keep it here.

Reply:
I think we won't need this file anymore. The legacy `atomicLong` codecs shouldn't be changed and even if they are changed we don't support any kind of backward compatibility.

Reply:
I know it's not needed but still I want to keep it as a documentation of codecs.